lib/core/db_models/old/schema/staff_personal_detail_schema.dart

```diff
@@ -9,8 +9,8 @@ class StaffPersonalDetailSchema {
   static const String checks = "checks";
   static const String building = "building";
   static const String street = "street";
-  static const String county = "county";
-  static const String city = "city";
-  static const String country = "country";
-  static const String postalCode = "postal_code";
+  static const String houseNumber = "house_number";
```

Comment:
guessing this is a mistake and we shouldn't have an underscore between these two words/

Reply:
its a key name which is coming from api, the variable name does not have any underscore